lib/okta.go

```diff
@@ -36,6 +36,8 @@ type OktaClient struct {
 	SessionToken    string
 	Expiration      time.Time
 	OktaAwsSAMLUrl  string
+	CookieJar       *cookiejar.Jar
```

Comment:
How about using the `http.CookieJar` interface here instead of the plain type to make the code more flexible? https://golang.org/pkg/net/http/#CookieJar

Reply:
sure 2450b3a